[PLT-1917][PLT-1968][PLT-1971][PLT-1957][PLT-1956][PLT-1958] Improve upgrade flow (#740)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/api-reference.adoc

| false
| -

| * `private_helm_repo`* _boolean_

> | Specifies the type of installation. If this field is enabled, charts to be downloaded during the cluster installation process will be retrieved from the repository specified as _helm_repository_.

Reviewer: ```suggestion
| Specifies the type of installation. If enabled, the charts required for the cluster installation will be downloaded from the repository specified as _helm++_++repository_.
```
